### DISCUSSION TITLE
Computer License -- Mutliple VMs on the same machine?

### DISCUSSION THREAD

**Discussion by S4lt5** [Vector35/binaryninja-api#2293] [Q&A] [answered]
Hi,

I just got a BN Single Computer license and it's currently installed on a linux VM for linux tasks. I also have a windows VM on the same box for windows tasks. Am I going to be OK installing it on both VMs on the same physical workstation?

**Comment by plafosse** [comment, marked as answer]
Yes that is fine.

**Reply by psifertex** [reply]
To be clear, our standard [named license](https://docs.binary.ninja/about/license.html#commercial-license-named) (which comes in both non-commercial and commercial varieties), allows you to install the software on multiple computers. The only restriction is that you must be the only person using it, and only one copy should be running at a time.
